rdrf/rdrf/templates/rdrf_cdes/alert_language_is_not_supported_message.html

```diff
@@ -0,0 +1 @@
+<div id="id_alert_language_is_not_supported_message">Language "%s (English: %s)" is not currently supported.</div>
```

Comment:
Is there any advantage to this being in it's own file? As far as I can tell, it is only referenced from one location.

Reply:
https://github.com/eresearchqut/trrf/pull/656#discussion_r1429350838